src/components/ens-profile/ActionButtons/MoreButton.tsx

```diff
@@ -160,30 +158,12 @@ export default function MoreButton({
     ]
   );
 
-  const handleAndroidPress = useCallback(() => {
-    const actionSheetOptions = menuItems
-      .map(item => item?.actionTitle)
-      .filter(Boolean) as any;
-
-    showActionSheetWithOptions(
-      {
-        options: actionSheetOptions,
-      },
-      async (buttonIndex: number) => {
-        // android: filter out undefined "Open Wallet" when not available
-        // otherwise it messes up the action order index
-        const items = menuItems.filter(item => item);
-        const actionKey = items[buttonIndex]?.actionKey;
-        handlePressMenuItem({ nativeEvent: { actionKey } });
-      }
-    );
-  }, [handlePressMenuItem, menuItems]);
-
+  const menuConfig = useMemo(() => ({ menuItems }), [menuItems]);
   return (
     <ContextMenuButton
       enableContextMenu
-      menuConfig={{ menuItems, menuTitle: '' }}
-      {...(android ? { onPress: handleAndroidPress } : {})}
+      menuConfig={menuConfig}
+      {...(android ? { handlePressMenuItem } : {})}
```

Comment:
anotha 1, why we passing this twice?

Reply:
no, this. is a different prop name. This will be unified later

Reply:
sry i dont know how to read